data/presets/man_made/flare.json

```diff
@@ -0,0 +1,25 @@
+{
+    "icon": "temaki-chimney",
+    "fields": [
+        "height"
```

Comment:
The wiki talks about https://wiki.openstreetmap.org/wiki/Key:structure but the description does not fit well IMO. What are your thoughts?

Reply:
Looked at a few examples and still don't see how the structure key adds value, so I close this.